rabbitmq/json-exchanges.yml

```diff
@@ -0,0 +1,123 @@
+type: json
+data:
+  url: /api/exchanges
+  server: ${SPM_MONITOR_RABBITMQ_API_URL}
+  basicHttpAuthUsername: ${SPM_MONITOR_RABBITMQ_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${SPM_MONITOR_RABBITMQ_BASICAUTH_PASSWORD}
+
+observation:
+  - name: exchanges
+    metricNamespace: rabbitmq
+    path: $.[?(@.name=${name} && @.vhost=${vhost})]
+
+    metric:
+      - name: exchanges.message_stats.ack
+        source: message_stats/ack
+        type: long_gauge
+        label: ack messages
+        description: Number of messages delivered to clients and acknowledged
+
+      - name: exchanges.message_stats.ack.rate
```

Comment:
Rate can be calculated in reports using formula.  Reading rate directly from the application may not work fine with our reports.

Same comment for all rate-related metrics.

Reply:
I will remove all rate metrics